WordPress/Classes/ViewRelated/NUX/SigninSelfHostedViewController.swift

```diff
@@ -4,9 +4,8 @@ import WordPressShared
 /// Provides a form and functionality to sign-in and add an existing self-hosted 
 /// site to the app.
 ///
-@objc class SigninSelfHostedViewController : NUXAbstractViewController, SigninKeyboardResponder
+@objc class SigninSelfHostedViewController : NUXAbstractViewController, SigninKeyboardResponder, SigninWPComSyncHandler
```

Comment:
> And yet, we have a SelfHosted controller that conforms to this protocol.

Maybe its just naming confusion? If the self-hosted form can be used to sign into a wpcom blog then it makes sense to conform to the sync handler, even tho its not the controller's intended purpose.

> it'd be interesting to update the credentials.

Its not that we don't want the credentials saved for self-hosted sites, but that we're only offering the option to sign in with saved credentials to wpcom sites.